src/pdf/ShippingLabels/ShippingLabelsPdf.tsx

```diff
@@ -82,70 +89,90 @@ interface LabelCardProps {
     quantity: number;
 }
 
+const convertDateFormat = (date: string): string => {
+    const dateArray = date.split("-");
+    return `${dateArray[2]}/${dateArray[1]}/${dateArray[0]}`;
+};
+
 const SingleLabelCard: React.FC<LabelCardProps> = ({ data, index, quantity }) => {
     return (
-        <Page size={LABEL_SIZE_PIXELS} style={styles.page}>
-            <View style={styles.cardWrapper} wrap={true}>
-                <View style={[styles.leftCol, { flexDirection: "column", marginRight: "3px" }]}>
-                    <View style={styles.firstRow}>
-                        <Text style={[styles.headingText, { flexWrap: "wrap" }]}>Name: </Text>
-                    </View>
-                    <View style={[styles.secondRow, { marginBottom: "10px" }]}>
-                        <Text style={[styles.fullNameText, { flexWrap: "wrap" }]}>
-                            {data.full_name}
-                        </Text>
-                    </View>
-                    <View style={styles.thirdRow}>
-                        {data.address_postcode && (
-                            <>
-                                <Text>
-                                    {data.address_1}
-                                    <br />
+        data.full_name !== "Deleted Client" && (
```

Comment:
It's best practice to not have conditional rendering at the top level of a component. Ideally what we'd do is move this check outside, to wherever this component is called.